documentation/slate/source/includes/realtime.md

```diff
@@ -0,0 +1,263 @@
+            "duration": 21180,
+            "from": ⊕{5 items},
+            "geojson": ⊕{3 items},
+            "id": "section_0_0",
+            "links": ⊕[6 items],
+            "stop_date_times": ⊖[
+               ⊕{7 items},
+               ⊖{
+                    "additional_informations": [],
+                    "arrival_date_time": "20190605T194700",
+                    "base_arrival_date_time": "20190605T193200",
+                    "base_departure_date_time": "20190605T193400"
+                    "departure_date_time": "20190605T194900",
+                    "links": [],
+                    "stop_point": ⊕{7 items},
+                },
+               ⊕{7 items},
+               ⊕{7 items},
+            ]
+            "to": ⊕{5 items},
+            "type": "public_transport",
+        }
+    ]
+```
+
+
+The status of the journey is `SIGNIFICANT_DELAYS`. In a public transport section of the response, "base_arrival_date_time"/"base_departure_date_time" represent the scheduled arrival/departure time without taking into account the delay whereas "arrival_date_time"/"departure_date_time" are the actual arrival/departure time, after the delay is applied. The delay can also be observed for every stop_point of the journey with the same parameters in "stop_date_times".
+If the parameter "data_freshness" is set to "base_schedule",  "base_arrival_date_time"/"base_departure_date_time" = "arrival_date_time"/"departure_date_time".
+
+A list of the disruptions impacting the journey is also present at the root level of the response.
+
+<aside class="notice">
+For a disruption to be present in the response, the request has to be made during its application period.
+</aside>
+
+<div></div>
+### Departures
+
+``` shell
+# Request example for /departures
+http://api.navitia.io/v1/coverage/<coverage>/physical_modes/<physical_mode>/stop_points/<stop_point>/departures?from_datetime=<from_date>&data_freshness=realtime
+```
+
+``` shell
+# Extract of an impacted departure object of the response /departures
+
+{
+    "display_informations": ⊕{13 items},
+    "links": ⊕[6 items],
+    "route": ⊕{9 items},
+    "stop_date_time": ⊖{
+        "additional_informations": [],
+        "arrival_date_time": "20190605T194700",
+        "stop_date_time": "realtime"
+        "departure_date_time": "20190605T194900",
+        "links": [],
+    }
+    "stop_point": ⊕{11 items},
+}
+```
+
+In the "stop_date_time" section of the response, the parameter "stop_date_time" is "realtime" and the fields "arrival_date_time"/"departure_date_time" take the delay into account.
+
+A list of the disruptions impacting the departures is also present at the root level of the response.
+
+<div></div>
+### Stop Schedules
+
+``` shell
+# Request example for /stop_schedules
+http://api.navitia.io/v1/coverage/<coverage>/physical_modes/<physical_mode>/lines/<line>/stop_points/<stop_point>/stop_schedules?from_datetime=<from_date>&data_freshness=realtime
```

Comment:
Please switch the example call to stop_area (to be consistent with trip-add and stop_add when they will be documented)